### PR TITLE
zh-CN:library.md

### PR DESCRIPTION
**Pull request by cnDengyu**

### CODE REVIEW

File: site/zh-cn/lite/microcontrollers/library.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# 理解 C++ 库
+
+微控制器版 TensorFlow Lite C++ 库是
+[TensorFlow 存储库](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro)的一部分。
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
[TensorFlow 仓库](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro)的一部分。
```

File: site/zh-cn/lite/microcontrollers/library.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+它可读、易修改，测试结果良好，易整合，
+并且与常规 TensorFlow Lite 兼容。
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
并且与标准 TensorFlow Lite 兼容。
```

File: site/zh-cn/lite/microcontrollers/library.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+在
+[README.md](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/README.md#how-to-port-tensorflow-lite-micro-to-a-new-platform)
+中包含更多关于所有这些话题的更多深入信息。
+
+## 文件结构
+
+在
+[`micro`](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro)
+根目录中有一个相对简单的结构。然而，因为它位于海量的 TensorFlow 存储库中，
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
根目录中有一个相对简单的结构。然而，因为它位于巨大的 TensorFlow 仓库中，
```

File: site/zh-cn/lite/microcontrollers/library.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+最重要的文件在项目的根目录中，并附带测试：
+
+-   [`all_ops_resolver.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/kernels/all_ops_resolver.h)
+    提供解释器运行模型的运算符。
+-   [`micro_error_reporter.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/micro_error_reporter.h)
+    输出调试信息。
+-   [`micro_interpreter.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/micro_interpreter.h)
+    包含控制和运行模型的代码
+
+参阅 [开始使用微控制器](get_started.md)
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
在 [开始使用微控制器](get_started.md) 可以找到
```

File: site/zh-cn/lite/microcontrollers/library.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+-   [`all_ops_resolver.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/kernels/all_ops_resolver.h)
+    提供解释器运行模型的运算符。
+-   [`micro_error_reporter.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/micro_error_reporter.h)
+    输出调试信息。
+-   [`micro_interpreter.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/micro_interpreter.h)
+    包含控制和运行模型的代码
+
+参阅 [开始使用微控制器](get_started.md) 
+以练习典型的用途。
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
典型的用途的展示。
```

File: site/zh-cn/lite/microcontrollers/library.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+要构建库并从主 TensorFlow 存储库中运行测试，
+请执行以下命令：
+
+1.  从 GitHub 中把 TensorFlow 存储库克隆到方便的地方。
+
+    ```bash
+    git clone --depth 1 https://github.com/tensorflow/tensorflow.git
+    ```
+
+2.  进入上一步创建的目录中。
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
2.  进入上一步创建的目录。
```

File: site/zh-cn/lite/microcontrollers/library.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+[README.md](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/README.md#how-to-port-tensorflow-lite-micro-to-a-new-platform)
+中包含更多关于所有这些话题的更多深入信息。
+
+## 文件结构
+
+在
+[`micro`](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro)
+根目录中有一个相对简单的结构。然而，因为它位于海量的 TensorFlow 存储库中，
+所以我们创建了一些脚本和预生成的项目文件，
+这些项目文件中分别提供了多种嵌入式开发环境
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
为多种嵌入式开发环境
```

File: site/zh-cn/lite/microcontrollers/library.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+在
+[README.md](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/README.md#how-to-port-tensorflow-lite-micro-to-a-new-platform)
+中包含更多关于所有这些话题的更多深入信息。
+
+## 文件结构
+
+在
+[`micro`](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro)
+根目录中有一个相对简单的结构。然而，因为它位于海量的 TensorFlow 存储库中，
+所以我们创建了一些脚本和预生成的项目文件，
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
所以我们创建了一些脚本和预生成的项目文件
```

File: site/zh-cn/lite/microcontrollers/library.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+这些项目文件中分别提供了多种嵌入式开发环境
+（如 Arduino, Keil, Make 和 Mbed）
+的相关源文件。
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
提供分离的相关源文件。
```

**Comment by cnDengyu** [comment]
I consent.

**Comment by MofiiTech** [comment]
LGTM

File: site/zh-cn/lite/microcontrollers/library.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
+    ```bash
+    make -f tensorflow/lite/experimental/micro/tools/make/Makefile test
+    ```
+
+## 写入新设备
+
+把微控制器版 TensorFlow Lite 写入新平台和设备的指南，
+可在
+[README.md](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro#how-to-port-tensorflow-lite-micro-to-a-new-platform)
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
[README.md](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro#如何将用于微控制器的TensorFlow-Lite写入一个新的平台)
```

File: site/zh-cn/lite/microcontrollers/library.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+[TensorFlow 仓库](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/experimental/micro)的一部分。
+它可读、易修改，测试结果良好，易整合，
+并且与标准 TensorFlow Lite 兼容。
+
+下面的文档列出了 C++ 库的基本结构，
+提供了编译所需的命令，并给出了将程序写入新设备
+的概览。
+
+在
+[README.md](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/README.md#how-to-port-tensorflow-lite-micro-to-a-new-platform)
```
Comment (MofiiTech) [inline site/zh-cn/lite/microcontrollers/library.md]:
```suggestion
[README.md](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/README.md#如何将用于微控制器的TensorFlow-Lite写入一个新的平台)
```

**Comment by MofiiTech** [comment]
LGTM

**Comment by MofiiTech** [comment]
I consent.

**Comment by Mr-Linus** [comment]
@cnDengyu You need to sign the CLA first.
If the CLA passed, this PR would be merged.

**Comment by MofiiTech** [comment]
@Mr-Linus cla/google said CLAs are signed but unable to verify author consent. Is it because I replied ‘I consent.’ too late that failed the checks? I don’t think there is a way to run the checks again and this may need the code owner to handle this?

**Comment by lamberta** [comment]
@cnDengyu The CLA bot has having difficulty because the email used to sign the CLA was not the same qq.com email used in the git commits.

**Comment by MofiiTech** [comment]
@lamberta Thanks for solving the issues!

**Pull request merged**